test/views/issueish-timeline-view.test.js

```diff
@@ -109,7 +109,7 @@ describe('IssueishTimelineView', function() {
     const wrapper = shallow(buildApp({
       relayHasMore: () => false,
     }));
-    assert.isFalse(wrapper.find('.github-PrTimeline-load-more-link').exists());
+    assert.isFalse(wrapper.find('.github-PrTimeline-loadMoreButton').exists());
```

Comment:
Not sure about the appropriate convention here, just noticing that we've switched from hyphen-separated to camelCase... same below.

Reply:
I think @simurai made that change - I'm happy to switch to kabob-case for the sake of consistency.

Reply:
The [naming convention](https://github.com/atom/github/blob/master/docs/naming-convention.md#child-element) *would* be camelCase for child elements of a component. So it can easily be read in groups: `github-PrTimeline-loadMoreButton` -> It's the `loadMoreButton` inside the `PrTimeline` component inside the `github` package. We don't follow that convention everywhere, which isn't a big deal.